crates/rustfix/tests/parse_and_replace.rs

```diff
@@ -6,65 +6,28 @@
 //! rustfix and applies the recommended suggestions to the `.rs` file. It then
 //! compares the result with the corresponding `.fixed.rs` file. If they don't
 //! match, then the test fails.
-//!
```

Comment:
Can we add a doc comment here for how to use the `SNAPSHOTS` env var? Something like 

```suggestion
//! To override snapshots, run `SNAPSHOTS=overwrite cargo test`.
//! See [`snapbox::assert::Action`] for more different actions.
```